frontstage/views/register/activate_account.py

```diff
@@ -21,13 +21,14 @@ def register_activate_account(token):
     except ApiError as exc:
         # Handle api errors
         if exc.status_code == 409:
-            logger.info('Expired email verification token', token=token)
+            logger.info('Expired email verification token', token=token, url=exc.url, status_code=exc.status_code)
```

Comment:
I think we should use use `api_url` and `api_status_code` in any place where we're logging the url and status code of the API we're calling. This will allow us to easily differentiate between this services errors and the API errors in the logs.